inputFiles/compositionalMultiphaseFlow/benchmarks/SPE10/SPE10.ats

```diff
@@ -0,0 +1,19 @@
+import geos_ats
```

Comment:
So are you adding new tests? Or are you basically "removing" the links with the file they are pointing to?

Reply:
I'm just about ready to start testing on the CI.  The last steps are to figure out how we want to fetch the baselines and the default path for things on the CI.

I'm not adding/removing any tests here.  Essentially, the script that I wrote did the following:

1) Find any .ats files in the integrated tests repo
2) Find any corresponding .xml file links in the same repo, resolving their absolute paths
3) For each set of (absolute) test directories, create a new test file with correct xml names (or a flag to indicate that a file isn't present in the target directory)

This was followed by some manual cleanup of invalid tests.

Reply:
We 100% should go through and further curate the test files